### PR TITLE
Add default/example policies to the docs

### LINKED ISSUE #244: Add default policies to fideslang/fidesctl
An additional way to reduce friction for users would be to have various "default" policies that users could choose from, for example:

1. GDPR
2. No Advertising
3. System Data Only
4. CCPA

This would allow engineers to only worry about their own dataset/system declarations and leave the legal/compliance side to a sensible default policy

This feature should allow users to define which default policies they want to use, if any

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
Closes #244 

### Code Changes

* [x] add a docs page with example policies
* [x] point `fides init` to the docs page (wait for #313)

### Steps to Confirm

* [ ] run `make docs-serve` and double check the page
* [ ] verify the policy definitions

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* [x] Documentation Updated

### Description Of Changes

Added some default policies to the docs and added a pointer to it in the `init` command. This should help people get started!

### CODE REVIEW

File: docs/fides/docs/guides/policies.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Example Policies
+
+This page gives a few examples of different policies one might want to use in their own organizations.
+
+## No Third-Party Data Sharing
+
+```yaml title="policy.yml"
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:7]: Let's promote good manifest file hygiene:
```suggestion
```yaml title="data_sharing_policy.yml"
```

File: docs/fides/docs/guides/policies.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+policy:
+  - fides_key: data_sharing_policy
+    name: Data Sharing Policy
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:10]: I think "policy" is redundant here.

```suggestion
    name: Data Sharing
```

File: docs/fides/docs/guides/policies.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          matches: ANY # And the data subject is a customer
+          values:
+            - customer
+        # And the data is identifiable, trigger a violation
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+```
+
+## Respecting Employee Data Privacy
+
+```yaml title="policy.yml"
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:34]:
```suggestion
```yaml title="employee_data_processing_policy.yml"
```

File: docs/fides/docs/guides/policies.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            - user
+        data_uses:
+          matches: ANY # And the use of the data is for third-party sharing
+          values:
+            - third_party_sharing
+        data_subjects:
+          matches: ANY # And the data subject is a customer
+          values:
+            - customer
+        # And the data is identifiable, trigger a violation
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:28]:
```suggestion
        # And the data is identifiable
```

File: docs/fides/docs/guides/policies.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+```
+
+## Respecting Employee Data Privacy
+
+```yaml title="policy.yml"
+policy:
+  - fides_key: employee_data_processing
+    name: Employee Data Processing
+    description: Restrict Employee Data Processing to only what is required for systematic business functions.
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:38]:
```suggestion
    description: Restrict employee data processing only to that which is required for systematic business functions.
```

File: docs/fides/docs/guides/policies.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+```yaml title="policy.yml"
+policy:
+  - fides_key: data_sharing_policy
+    name: Data Sharing Policy
+    description: The privacy policy that governs sharing of data with third parties.
+    rules:
+      - name: Disallow Third-Party Marketing
+        description: Disallow collecting any user contact info to use for marketing.
+        data_categories:
+          matches: ANY # If any of these data categories are being used
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:16]:
```suggestion
          matches: ANY # Trigger a violation if any of these data categories are being used
```

File: docs/fides/docs/guides/policies.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+```yaml title="policy.yml"
+policy:
+  - fides_key: employee_data_processing
+    name: Employee Data Processing
+    description: Restrict Employee Data Processing to only what is required for systematic business functions.
+    rules:
+      - name: Disallow Non-System Use of Employee Data
+        data_categories:
+          matches: ANY # If any of these data categories are being used
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:42]:
```suggestion
          matches: ANY # Trigger a violation if any of these data categories are being used
```

File: docs/fides/docs/guides/policies.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+        data_uses:
+          matches: OTHER # And the use of the data is something other than...
+          values:
+            - provide.system
+            - improve.system
+        data_subjects:
+          matches: ANY # And the data subject is an employee
+          values:
+            - employee
+        # And the data is identifiable, trigger a violation
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:55]:
```suggestion
        # And the data is identifiable
```

File: docs/fides/docs/guides/policies.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+```
+
+## Respecting Biometric PII
+
+```yaml title="policy.yml"
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:61]:
```suggestion
```yaml title="biometric_data_policy.yml"
```

File: docs/fides/docs/guides/policies.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+policy:
+  - fides_key: biometric_data_policy
+    name: Biometric Data Policy
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:64]:
```suggestion
    name: Biometric Data
```

File: docs/fides/docs/guides/policies.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+    description: Policy that describes valid uses of biometric and health data.
+    rules:
+      - name: Disallow Biometrics for Profit.
+        description: Disallow biometric data being used for profit-related use-cases.
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:68]:
```suggestion
        description: Disallow the use of biometric data for profit-related purposes.
```

File: docs/fides/docs/guides/policies.md [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+        data_categories:
+          matches: ANY # If any of these data categories are being used
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:70]:
```suggestion
          matches: ANY # Trigger a violation if any of these data categories are being used
```

File: docs/fides/docs/guides/policies.md [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+          values:
+            - advertising
+            - train_ai_system
+            - improve
+            - third_party_sharing
+        data_subjects:
+          matches: ANY # And the data subject is a customer
+          values:
+            - customer
+        # And the data is identifiable, trigger a violation
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:85]:
```suggestion
        # And the data is identifiable
```

File: docs/fides/docs/guides/policies.md [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+```
+
+## Anonymous Derived User Contact Data
+
+```yaml title="policy.yml"
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:91]:
```suggestion
```yaml title="protect_derived_user_data_policy.yml"
```

File: docs/fides/docs/guides/policies.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+policy:
+  - fides_key: protect_derived_user_data
+    name: Protect Derived User Data
+    description: Policy that describes the proper use of derived user data.
+    rules:
+      - name: Disallow Non-Anonymized Derived User Data.
+        description: Require that any use of derived user data must be de-identified to the anonymous level, as opposed to the pseudonymous.
+        data_categories:
+          matches: ANY # If any of these data categories are being used
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:100]:
```suggestion
          matches: ANY # Trigger a violation if any of these data categories are being used
```

File: docs/fides/docs/guides/policies.md [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+          values:
+            - user.derived.identifiable.biometric_health
+        data_uses:
+          matches: NONE # And for any data use
+          values: []
+        data_subjects:
+          matches: ANY # And the data subject is a customer
+          values:
+            - customer
+        # And the data is either unlinked_pseudonymized or more identifiable, trigger a violation
```
Comment (PSalant726) [inline docs/fides/docs/guides/policies.md:110]:
```suggestion
        # And the data is either unlinked_pseudonymized or more identifiable
```

File: fidesctl/.fides/policy.yml [hunk 17]
```diff
@@ -18,3 +18,25 @@ policy:
...
             - anonymous_user
         data_qualifier: aggregated
+  - fides_key: data_sharing_policy
+    name: Data Sharing Policy
+    description: The privacy policy that governs sharing of data with third parties.
+    rules:
+      - name: Disallow Third-Party Marketing
+        description: Disallow collecting any user contact info to use for marketing.
+        data_categories:
+          matches: ANY # If any of these data categories are being used
```
Comment (PSalant726) [inline fidesctl/.fides/policy.yml:28]:
```suggestion
          matches: ANY # Trigger a violation if any of these data categories are being used
```

File: fidesctl/.fides/policy.yml [hunk 18]
```diff
@@ -18,3 +18,25 @@ policy:
...
+            - user.provided.identifiable
+        data_uses: # And the use of the data is for third-party sharing
+          matches: ANY
+          values:
+            - third_party_sharing
+        data_subjects:
+          matches: ANY # And the data subject is customer data
+          values:
+            - customer
+        # And the data is identifiable, trigger a violation
```
Comment (PSalant726) [inline fidesctl/.fides/policy.yml:41]:
```suggestion
        # And the data is identifiable
```

File: fidesctl/src/fidesctl/cli/util_comands.py [hunk 19]
```diff
@@ -35,14 +35,14 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
         "cli": {"server_url"},
     }
 
-    print("Initializing Fidesctl...\n")
+    click.echo("Initializing Fidesctl...\n")
```
Comment (PSalant726) [inline fidesctl/src/fidesctl/cli/util_comands.py:38]: According to [the `click.echo` documentation](https://click.palletsprojects.com/en/7.x/utils/#printing-to-stdout), a newline is emitted by default.
```suggestion
    click.echo("Initializing Fidesctl...")
```

File: fidesctl/src/fidesctl/cli/util_comands.py [hunk 20]
```diff
@@ -35,14 +35,14 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
...
 
     # create the dir if it doesn't exist
     if not os.path.exists(fides_dir_path):
         os.mkdir(fides_dir_path)
         echo_green(f"Created a '{fides_dir_path}' directory.\n")
     else:
-        print(f"Directory '{fides_dir_path}' already exists.\n")
+        click.echo(f"Directory '{fides_dir_path}' already exists.\n")
```
Comment (PSalant726) [inline fidesctl/src/fidesctl/cli/util_comands.py:45]:
```suggestion
        click.echo(f"Directory '{fides_dir_path}' already exists.")
```

File: fidesctl/src/fidesctl/cli/util_comands.py [hunk 21]
```diff
@@ -55,7 +55,11 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
         echo_green(config_message)
 
     else:
-        print(f"Configuration file already exists at '{config_path}'.\n")
+        click.echo(f"Configuration file already exists at '{config_path}'.\n")
```
Comment (PSalant726) [inline fidesctl/src/fidesctl/cli/util_comands.py:58]:
```suggestion
        click.echo(f"Configuration file already exists at '{config_path}'.")
```

File: fidesctl/src/fidesctl/cli/util_comands.py [hunk 22]
```diff
@@ -55,7 +55,11 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
...
+
+    click.echo(
+        "For example policies to help get started, see: ethyca.github.io/fides/guides/policies/\n"
+    )
```
Comment (PSalant726) [inline fidesctl/src/fidesctl/cli/util_comands.py:62]:
```suggestion
    click.echo(
        "For example policies and help getting started, see: ethyca.github.io/fides/guides/policies/"
    )
```

**Review by PSalant726 — APPROVED** [review]
Lots of comments, but they're all just nitpicks. There's nothing blocking here.

File: docs/fides/docs/guides/policies.md [hunk 23]
```diff
@@ -0,0 +1,112 @@
...
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized
```
Comment (iamkelllly) [inline docs/fides/docs/guides/policies.md:111]: The definition of "Unlinked pseudonymized data" is data for which all identifiers are erased or substituted by aliases for which the assignment function *is erased or irreversible, such that the linkage cannot be re-established by reasonable efforts of anyone including the party that performed them*.
I would be included to say that "unlinked pseudonymized" is an acceptable level of deidentifiability under the EDPB ruling. however, vanilla `pseudo` or `identified` is unacceptable, so maybe this should be `data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized`?

**Review by iamkelllly — CHANGES_REQUESTED** [review]
Thanks for doing this @ThomasLaPiana ! First one looks good, nice and simple; i had a few comments on the other 3. 
While you're in here :trollface: would you also mind adding a 5th for the TCPA, which I left in a [comment here](https://github.com/ethyca/fides/issues/150#issuecomment-998335264) for allowing phone numbers for transactional messaging only?

**Comment by ThomasLaPiana** [comment]
@iamkelllly thank you so much for the feedback, integrated those comments and its ready for another review

File: fidesctl/src/fidesctl/cli/util_comands.py [hunk 24]
```diff
@@ -23,6 +23,7 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
     """
 
     # Constants
+    separate = lambda: print("-" * 15)
```
Comment (PSalant726) [inline fidesctl/src/fidesctl/cli/util_comands.py:26]: Not a fan, personally.

**Review by PSalant726 — APPROVED** [review]

File: fidesctl/src/fidesctl/cli/util_comands.py (hunk 24 above)
Comment (ThomasLaPiana) [reply fidesctl/src/fidesctl/cli/util_comands.py:26]: this gives it a consistent look with apply/evaluate

**Review by iamkelllly — APPROVED** [review]
thanks for making these updates! lgtm 👍

**Pull request merged**